server/channels/app/platform/web_broadcast_hook.go

```diff
@@ -29,7 +29,10 @@ func (h *Hub) runBroadcastHooks(msg *model.WebSocketEvent, webConn *WebConn, hoo
 			continue
 		}
 
-		hook.Process(hookedEvent, webConn, args)
+		err := hook.Process(hookedEvent, webConn, args)
+		if err != nil {
+			mlog.Error("runBroadcastHooks: Error processing hook", mlog.String("hook_id", hookID), mlog.Err(err))
```

Comment:
Let's lower this to a warning
```suggestion
			mlog.Warn("runBroadcastHooks: Error processing hook", mlog.String("hook_id", hookID), mlog.Err(err))
```

Reply:
@hanzei 
Thank you for your feedback.
I updated the code.